.github/workflows/nightly.yaml

```diff
@@ -8,38 +7,27 @@ on:
       - main
 
 env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   INTEGRATION: "prometheus"
   ORIGINAL_REPO_NAME: 'newrelic/nri-prometheus'
-  REPO_FULL_NAME: ${{ github.event.repository.full_name }}
   TAG: nightly
-  TAG_SUFFIX: "-nightly" # In order to avoid nightly updating images
+  TAG_SUFFIX: "-nightly"
 
 jobs:
-  release:
-    name: Build and release nightlies with goreleaser
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Set up Go
-        uses: actions/setup-go@v5
-        with:
-          go-version-file: 'go.mod'
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-      - name: Login to DockerHub
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
-          password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
-      - name: Do the release
-        # The release target is not being executed inside a container (ci/release) since the target
-        # compiles docker images (from goreleaser) and that cannot be done inside a container.
-        run: make release
-        env:
-          GENERATE_PACKAGES: true
+  nightly:
+    uses: newrelic/coreint-automation/.github/workflows/reusable_nightly.yaml@v3
+    secrets:
+      docker_username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
+      docker_password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
+    with:
+      docker_image: newrelic/nri-prometheus
+      docker_tag: nightly
+      target_branches: "main"
+      integration_name: "prometheus"
+      build_command: |
+        make release
```

Comment:
Why do we need it to be multiline when we just have a single line of input?

Reply:
That was left over from copying configs from repos where multiline command were used.

Reply:
Can we update it?